src/pages/payments/refund/Refund.tsx

```diff
@@ -0,0 +1,213 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @copyright Copyright (c) 2021. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @copyright Copyright (c) 2021. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { Card, Element } from '@invoiceninja/cards';
+import { Button, InputField, SelectField } from '@invoiceninja/forms';
+import axios, { AxiosError } from 'axios';
+import { endpoint } from 'common/helpers';
+import { Invoice } from 'common/interfaces/invoice';
+import { ValidationBag } from 'common/interfaces/validation-bag';
+import { defaultHeaders } from 'common/queries/common/headers';
+import { usePaymentQuery } from 'common/queries/payments';
+import { Alert } from 'components/Alert';
+import Toggle from 'components/forms/Toggle';
+import { useFormik } from 'formik';
+import { useEffect, useState } from 'react';
+import toast from 'react-hot-toast';
+import { useTranslation } from 'react-i18next';
+import { useQueryClient } from 'react-query';
+import { generatePath, useParams } from 'react-router-dom';
+
+export function Refund() {
+  const queryClient = useQueryClient();
+  const { id } = useParams();
+  const [t] = useTranslation();
+  const { data: payment } = usePaymentQuery({ id });
+  const [errors, setErrors] = useState<ValidationBag>();
+  const [invoices, setinvoices] = useState<string[]>([]);
```

Comment:
Conventions are off. I mentioned this in the previous PR. Please follow the conventions for the project.

Variables are camel case.
Functions, classes are pascal case.
Files are kebab cases, except hooks. They use pascal case as well.